extensions/wikia/Email/styles/userNameLayout.css

```diff
@@ -0,0 +1,36 @@
+.avatar-wrapper {
+	margin: 20px 0;
+	padding-right: 30px;
+}
+
+.avatar {
+	border-radius: 100%;
+}
+
+.avatar img {
+	height: 50px;
+	width: 50px;
+}
+
+.border {
+	border-top: 1px solid #e6ebf2;
+	border-bottom: 1px solid #e6ebf2;
+}
+
+.greeting p:last-child {
+	font-weight: bold;
+}
+
+.username {
+	font-size: 12px;
+}
+
+.message {
+	margin: 20px 0;
+}
+
+.message p {
```

Comment:
It's a small thing, but I think our coding conventions call for css/scss to be ordered alphabetically. Can we move these 2 `.message` rules up above `.username`?